app/src/main/res/drawable/bg_post_item.xml

```diff
@@ -0,0 +1,8 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="#EDE9C5" />

```

Comment:
Extract this value into the `dimens.xml` file

Reply:
done